Library/Homebrew/extend/ENV/std.rb

```diff
@@ -149,11 +149,6 @@ def gcc
     end
   end
 
-  def llvm
-    super
-    set_cpu_cflags
-  end
```

Comment:
Any reason why `MacOS.llvm_build_version` is retained but `ENV.llvm` (this here) can be safely dropped? (Or should the latter be moved to `compat/` instead?)